src/main/java/com/gmail/nossr50/database/FlatfileDatabaseManager.java

```diff
@@ -1148,7 +1146,7 @@ public int compare(PlayerStat o1, PlayerStat o2) {
         }
     }
 
-    private PlayerProfile loadFromLine(@NotNull String[] character, boolean updateRequired) {
+     private PlayerProfile loadFromLine(String[] character) {
```

Comment:
I used IntelliJ's refactor signature tool and it looks like it deleted the annotation in the process, so yeah it would be a mistake, it will be added back in.